api/openapi.gen.json

```diff
@@ -33,6 +33,48 @@
       }
     },
     "schemas": {
+      "v1.AWSReservation": {
+        "properties": {
+          "amount": {
+            "format": "int32",
+            "type": "integer"
+          },
+          "aws_reservation_id": {
+            "type": "string"
+          },
+          "created_at": {
+            "format": "date-time",
+            "type": "string"
+          },
+          "finished_at": {},
+          "id": {
+            "format": "int64",
+            "type": "integer"
+          },
+          "image_id": {
+            "type": "string"
+          },
+          "instance_type": {
+            "type": "string"
+          },
+          "provider": {
+            "type": "integer"
+          },
+          "pubkey_id": {
+            "format": "int64",
+            "type": "integer"
+          },
+          "source_id": {
+            "format": "int64",
+            "type": "integer"
+          },
+          "status": {
+            "type": "string"
+          },
+          "success": {}
```

Comment:
I guess the same goes for success. @lzap ?

Reply:
I have changed it but it doesnt seem to work for noop and was, do you know why?
I tried to remove the usage of &models.SOME_MODEL but that all the fields were removed for some reason

Reply:
Found the reason, you embed the models into payloads. They do overwrite the fields with the sql.Null ones. You should not use embedding, just copy and paste those fields and make sure you leave only json tags in payloads and db tags in the model:

```diff
        // Flag indicating success, error or unknown state (NULL). See Status for the actual error.
-       Success sql.NullBool `db:"success" json:"success"`
+       Success sql.NullBool `db:"success"`
 }
```